ui/hooks/useMultichainSelector.ts

```diff
@@ -7,7 +7,7 @@ export function useMultichainSelector<
   TSelected = unknown,
 >(
   selector: (state: TState, account?: InternalAccount) => TSelected,
-  account?: InternalAccount,
+  account?: InternalAccount | null,
```

Comment:
Some hooks return null for the account. This just makes it easier to pass the value.